build_tools/scripts/bazel_to_cmake.py

```diff
@@ -95,14 +97,30 @@ def _convert_name_block(self, **kwargs):
     #    rule_name
     return "  NAME\n    %s\n" % (kwargs["name"])
 
-  def _convert_namespace_block(self, **kwargs):
+  def _convert_cc_namespace_block(self, **kwargs):
     #  CC_NAMESPACE
     #    "cc_namespace"
     return "  CC_NAMESPACE\n    \"%s\"\n" % (kwargs["cc_namespace"])
 
+  def _convert_cpp_namespace_block(self, **kwargs):
+    #  CPP_NAMESPACE
+    #    "cpp_namespace"
+    return "  CPP_NAMESPACE\n    \"%s\"\n" % (kwargs["cpp_namespace"])
+
   def _convert_translation_block(self, **kwargs):
     return "  TRANSLATION\n    \"%s\"\n" % (kwargs["translation"])
 
+  def _convert_translate_tool_block(self, **kwargs):
+    translate_tool = kwargs.get("translate_tool")
+    print(translate_tool)
```

Comment:
remove `print()`?

Reply:
Just a left over... Thanks for catching this!